packages/core/src/lib/wallet/wallet.types.ts

```diff
@@ -121,6 +128,112 @@ interface SignAndSendTransactionsParams {
   transactions: Array<Optional<Transaction, "signerId">>;
 }
 
+interface SignTransactionParams {
+  /**
+   * The Transaction object to sign
+   */
+  transaction: Tx;
```

Comment:
App developer in most of the cases cannot construct the full transaction since there is no public key and therefore there is no nonce. Also, it is often no account id either.

I cannot think of the use for `SignTransactionParams` interface as it is defined right now.